content/community/changelog/app-frontend/stricter-schema/_index.en.md

```diff
@@ -6,9 +6,9 @@ toc: true
 ---
 
 # What?
-During the end of August 2023 we will upgrade the layout JsonSchema used by Altinn 3 apps to verify the layout files.
-This will make the JsonSchema stricter and might cause your app to display an increased amount of validation errors
-(usually yellow lines) when developing locally in Visual Studio Code (or similar IDEs).
+During the start of September 2023 upgraded the layout JsonSchema used by Altinn 3 apps to verify the layout files.
```

Comment:
"we upgraded" maybe?

Reply:
Whoops.. 🙄 I'll make note of this and fix it in my next PR - pushing straight to master was rejected..